hermes/probe/delete_file_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Evan Spendlove, GitHub: evanSpendlove.
+//
+// Delete_file_test tests the file deletion operation for MonitorProbe.
+// Package probe implements the probe that Hermes uses to monitor
+// a storage system.
+
+package probe
+
+import (
+	"context"
+	"testing"
+
+	"cloud.google.com/go/storage"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+)
+
+func TestDeleteRandomFile(t *testing.T) {
```

Comment:
Although there isn't support for NIL files, yet. I think it might be worth adding a test scenario to delete a nil file.

Reply:
Since there isn't any variable passed to identify the file to be deleted and the NIL file won't ever be selected by the number returned from DeletionPicker.go, I don't think we can add a test scenario of deleting a NIL file.

Reply:
As in, the NIL file won't be chosen to be deleted, and we can't modify the `DeleteRandomFile()` function inside the test to do this, so I'm not sure how we could test this.